### ISSUE TITLE
Question about userScoring in egs_application

### ISSUE THREAD

**Issue by MartinMartinov** [nrc-cnrc/EGSnrc#673]
I came across an issue when trying to score alphas generated in egs_radionuclide_source in my custom usercode, which seemed like a bug to me (though I might be wrong).  After the ausgab object scoring implementation, there's this section in EGS_Application::userScoring(int iarg, int ir):

`if (ir > -1) {  
    return 0;  
}  
else {  
    return ausgab(iarg);  
}`

which seems to only invoke ausgab when ir is -1 or lower, and ausgab doesn't get invoked if ir is an actual region in the phantom.

To me this seems like the return calls are reversed, and ausgab is supposed to be invoked when ir is in the geometry.  I haven't looked too deeply in the egs++ code beyond geometries, sources, and some advanced_application stuff, so I'm still a little unsure; some outside input would be very much appreciated :).

**Comment by MartinMartinov** [comment]
Maybe I should have titled this as a potential bug?

**Comment by rtownson** [comment]
Yes, looks like a bug to me and you're probably right about the fix! Sorry for the delayed response.

**Comment by MartinMartinov** [comment]
No problem, looking back I can see my title definitely was not the clearest :)
